syscalls.c

```diff
@@ -914,6 +914,30 @@ int syscalls_signalSuspend(void *ustack)
 }
 
 
+int syscalls_sigreturn(void *ustack)
```

Comment:
Always zero, we should kill the process imho in case of some shenanigans on it's part
```suggestion
void syscalls_sigreturn(void *ustack)
```